### ISSUE TITLE
ModSecurity: SecRemoteRules: ModSecurity was not compiled with SecRemoteRules support.

### ISSUE THREAD

**Issue by pislisSniflis** [SpiderLabs/ModSecurity#1221]
Hi, I bought SLR 2 days ago, but until today I can't use the rules because of issue on ModSec setup.

I have been guided by the team to uninstall and install the latest version of 2.9.1 but at the end I face the above error.

Can someone help me?

The technical people ask me to download a few package / library that requires by SecRemoteRules; but actually I have downloaded it before.

Since the problem happen since 3 days; my client not very happy. I try to find the solution from the internet but still I failed. Hope you guys can help me on specific way on enabling the secremoterules module

**Comment by zimmerle** [comment]
Hi @pislisSniflis,

I am assuming that the error that you are facing is the one described on the issue title. If so, it is happening because you have compiled ModSecurity without libcurl dependence. Libcurl is mandatory to enable the SecRemoteRules command. Do you have the libcurl-dev package installed on your distro?

**Comment by pislisSniflis** [comment]
Yes, I have it. I recompiled the installation but still not solved. 
Finally I installed using package provided by Ubuntu 16.04 and the problem solved. In the first place, I have been advised to reinstall modsec from source, because my actual problem is I can't download the file using secremoterules module (with modsec version 2.9.0 - package from Ubuntu). I remove the package and using source installation antoher problem happen which it's not enable the secremoterules and I revert back to package from Ubuntu to solve the problem.
